client/my-sites/site-settings/index.js

```diff
@@ -111,6 +112,16 @@ export default function () {
 		clientRender
 	);
 
+	page(
+		'/settings/header-code/:site_id',
```

Comment:
If this isn't supported for the current site type (for example, WP.com simple) then we should make sure to redirect it away.

Reply:
This is still valid.

Reply:
Switched to ensure that unsupported sites are redirected instead.

Reply:
Thank you!